core/trace/src/main/scala/org/typelevel/otel4s/trace/SpanContext.scala

```diff
@@ -43,10 +46,9 @@ trait SpanContext {
     */
   def spanIdHex: String
 
-  /** Returns the sampling strategy of this [[SpanContext]]. Indicates whether
-    * the span in this context is sampled.
-    */
-  def samplingDecision: SamplingDecision
+  def traceFlags: TraceFlags
+
+  def isSampled: Boolean = traceFlags.isSampled
```

Comment:
There's some related work in @sherriesyt's PR, we might want to merge that in.
- https://github.com/typelevel/otel4s/pull/236

Reply:
Yeah. I guess we should incorporate the changes from #236 into this PR.